agent/tests/test_txn.c

```diff
@@ -161,6 +162,41 @@ static void test_max_segments_config_values(TSRMLS_D) {
   tlib_php_request_end();
 }
 
+
+static void test_set_initial_path(TSRMLS_D) {
+  nrtxn_t* txn;
+  zval* server;  
+  /*
+   * Test : max_segments_cli defaults correctly.
```

Comment:
This comment doesn't seem correct for this function - was it copied accidentally?

Reply:
I'm sorry. I fixed it.